docs/data/toolpad/getting-started/introduction.md

```diff
@@ -0,0 +1,26 @@
+# Introduction
+
+## What is Toolpad?
+
+<p class="description">
+Quickly build internal tools. MUI Toolpad is a self-hosted low-code internal tools builder designed to extend the MUI React components. It's designed for developers of all trade who want to save time building internal applications.
+</p>
```

Comment:
The description needs to be right after the h1:

```suggestion

<p class="description">Quickly build internal tools. MUI Toolpad is a self-hosted low-code internal tools builder designed to extend the MUI React components. It's designed for developers of all trades who want to save time building internal applications.</p>

## What is Toolpad?
```

*x the other pages*

Reply:
Also, this description is too long. It needs to be between 110 and 160 characters.

Reply:
I took description from our repository, wondering which parts should we omit to make it shorter?

Maybe it would be enough to ditch the last sentence?

> Quickly build internal tools. MUI Toolpad is a self-hosted low-code internal tools builder designed to extend the MUI React components.